.github/workflows/language-reference.yaml

```diff
@@ -0,0 +1,61 @@
+name: language-reference-documentation
+
+on:
+  push:
+    branches:
+      - 'language-reference-stable'
+  pull_request:
+    branches:
+      - 'language-reference-stable'
+
+jobs:
+  build-and-push:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Git Checkout
+        uses: actions/checkout@v3
+
+      - name: Cache Coursier
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/coursier
+          key: sbt-coursier-cache
+
+      - name: Cache SBT
+        uses: actions/cache@v1
+        with:
+          path: ~/.sbt
+          key: sbt-${{ hashFiles('**/build.sbt') }}
```

Comment:
Do you think these could and should be replaced with https://github.com/coursier/cache-action?